routes/repos/[owner]/[repo]/readme.ts

```diff
@@ -6,12 +7,19 @@ export default cachedEventHandler(
     const defaultBranch = await ghRepo(repo).then(
       (r) => r.default_branch || "main",
     );
-    const cdnBaseURL = `${GH_RAW_URL}/${repo}/${defaultBranch}`;
-    const markdown = await $fetch<string>(`${cdnBaseURL}/README.md`);
+
+    const linkOptions = {
+      cdnBaseURL: `${GH_RAW_URL}/${repo}/${defaultBranch}`,
+      githubBaseURL: `${GH_PATH_URL}/${repo}/tree/${defaultBranch}`,
+    };
+    const markdown = await $fetch<string>(
+      `${linkOptions.cdnBaseURL}/README.md`,
+    );
     const html = await ghMarkdown(markdown, repo, "readme");
```

Comment:
We can probably run `resolveMarkdownRelativeLinks` once beforehand and pass the modified markdown to `ghMarkdown` 🤔